src/styles/sidebar/components/group-list-item-out-of-scope.scss

```diff
@@ -0,0 +1,66 @@
+/* The group-list out of scope item. */
+
+.group-list-item-out-of-scope {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+}
+
+.group-list-item-out-of-scope__item {
+  background-color: $gray-lightest;
+  color: $gray-light;
+}
+
+.group-list-item-out-of-scope__icon--unavailable {
+  fill: $gray-light;
+  float: right;
+  height: 20px;
+  width: auto;
+}
+
+.group-list-item-out-of-scope__details {
+  flex-grow: 1;
+  flex-shrink: 1;
+  font-weight: 500;
+
+  .group-list-item-out-of-scope__details--unavailable-message,
+  .group-list-item-out-of-scope__details--actions {
+    display: none;
+  }
+
+  &.expanded {
+    .group-list-item-out-of-scope__details--toggle {
+      display: none;
+    }
+
+    .group-list-item-out-of-scope__details--unavailable-message,
+    .group-list-item-out-of-scope__details--actions {
+      display: block;
```

Comment:
Rather than hiding and showing these elements with CSS, I think it will be clearer to hide/show them in the template using `ng-if` or `ng-show` for consistency, unless there is some reason that I missed why it is difficult to do in this context.